src/app/sync/sync.service.ts

```diff
@@ -109,6 +109,11 @@ export class SyncService {
         this._connections.next(connections);
       });
       this._socket.connect();
+      if (!this._state.getShowCurrentLocation()) return;
+      setTimeout(() => {
+        this._state.updateShowCurrentLocation(false);
+        this._state.updateShowCurrentLocation(true);
```

Comment:
Wieso wird hier hin und her getogglet?

Reply:
Falls die Socket-Connection reconnected wird, muss zuerst 
`        this._state.updateShowCurrentLocation(false);
`
ausgeführt werden, damit der Benutzer nicht doppelt erscheint.